app/src/main/java/com/google/android/samples/socialite/ui/videoedit/VideoEditScreenViewModel.kt

```diff
@@ -183,3 +187,9 @@ class VideoEditScreenViewModel @Inject constructor(
         }
     }
 }
+
+enum class VideoSaveState {
+    PENDING,
+    VIDEO_SAVE_SUCCESS,
+    VIDEO_SAVE_FAIL,
+}
```

Comment:
Same question as the comment left on the `enum class ImageCaptureState`.

Reply:
Per conversation thread on other comment, leaving this as enum, but I've addressed the `else` in the `when` statement.